sklearn/_build_utils/deprecated_modules.py

```diff
@@ -271,7 +271,7 @@
 _FILE_CONTENT_TEMPLATE = """
 # THIS FILE WAS AUTOMATICALLY GENERATED BY deprecated_modules.py
 import sys
-from . import {new_module_name}
+from . import {new_module_name}  # type: ignore
```

Comment:
These are often C extensions (which raise mypy type errors), and since this module is deprecated it doesn't matter anyway.